src/GitTools.Core.Tests/Extensions/StringExtensionsFacts.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace GitTools.Tests
+{
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class StringExtensionsFacts
+    {
+        [TestCase("/develop", false)]
+        [TestCase("/master", false)]
+        [TestCase("/pr/25", true)]
+        [TestCase("/pull/25", true)]
+        [TestCase("/pull-requests/25", true)]
+        public void TheIsPullRequestMethod(string input, bool expectedValue)
+        {
+            var actualValue = input.IsPullRequest();
+            
+            Assert.AreEqual(expectedValue, actualValue);
```

Comment:
I'll give you this one. It's all a matter of preference, to be honest I think nunit provides really good feedback (I expected x, but got y). Not sure how shouldly does it but I'll do it anyway.